test/tooling.test.ts

```diff
@@ -88,9 +88,9 @@ it('should run tests synchronously', async () => {
  *
  */
 it('should get completions and return completions', async () => {
-  const res = await conn.tooling.completions('apex');
+  const res = await conn.tooling.completions('visualforce');
```

Comment:
getting apex completions takes ~2min so this tests was timing out a lot, changed it to get VF completions.

tooling API:
https://developer.salesforce.com/docs/atlas.en-us.api_tooling.meta/api_tooling/intro_rest_resources.htm?q=completion+

response contains these components:
https://developer.salesforce.com/docs/atlas.en-us.pages.meta/pages/pages_compref.htm

`tooling.completions`:
https://github.com/jsforce/jsforce/blob/3.0/src/api/tooling.ts#L403-L415

Reply:
apex completions:
`sf org api 'services/data/v59.0/tooling/completions?type=apex' -o <org>` takes ~2min even after the first request

visualforce completions:
`sf org api 'services/data/v59.0/tooling/completions?type=visualforce' -o <org>` takes ~5sec.